include/internal/libspdm_common_lib.h

```diff
@@ -335,10 +366,14 @@ typedef struct {
     libspdm_message_b_managed_buffer_t message_mut_b;
     libspdm_message_c_managed_buffer_t message_mut_c;
     libspdm_message_m_managed_buffer_t message_m;
+    libspdm_message_e_managed_buffer_t message_e;
+    libspdm_message_e_managed_buffer_t message_mut_e;
```

Comment:
`_mut_` was used for mutual authentication, however `GET_ENDPOINT_INFO` cannot be issued during mutual authentication. Maybe `_encap_` is better suited? @jyao1 to comment.

Reply:
agree.